published/025-2019-jan-15.md

```diff
@@ -8,14 +8,22 @@ Want this update in your inbox? [Subscribe to our weekly newsletter!](https://ti
 
 Here are some of the highlights since the last IPFS Weekly.
 
-## The latest
 
-+ [Making the Web Work Offline - Molly Mackinlay](https://www.youtube.com/watch?list=PLuhRWgmPaHtRdiy0HKNy4UZ4dKVUVL_KG&time_continue=7&v=VllPpxVNH5k), plus chance to join the conversation at upcoming meetup Jan 23
+## Announcing ProtoSchool
+
+This week we’re excited to announce the launch of [ProtoSchool](https://proto.school/), an educational community that teaches decentralized web protocols and tools through online tutorials and local chapter events.
+
+[Self-guided interactive tutorials](https://proto.school/#/tutorials) teach new concepts through code challenges completed in the browser, with feedback to help you track your progress. Tutorials are built by community members like you with an interest in helping others learn about IPFS and other decentralized web projects such as IPLD, LibP2P, Multiformats, and Filecoin.
+
+[Local chapter events](https://proto.school/#/chapters) around the world use ProtoSchool tutorials as curriculum, with mentors available to help participants work through the challenges. ProtoSchool chapters operate independently, but each has its own repo in our org to facilitate chapter discussions and event planning, as well as easy hosting of a chapter website via GitHub Pages. [Start your own chapter today!](https://proto.school/#/host)
+
+
```

Comment:
@renrutnnej I was able to get some help editing yesterday's launch presentation into a 7.5 minute video that's now available on YouTube.  Would you like to add this at the end of this section? 

`Check out our [launch video](https://youtu.be/G5aiLkHZ7CM) to learn more about how you can get involved with the ProtoSchool community.`

Reply:
@terichadbourne I’ve already sent out the newsletter but I can add this bit to the blog and archival version.